test/coverlet.core.tests/Instrumentation/ModuleTrackerTemplateTests.cs

```diff
@@ -9,62 +12,83 @@ namespace Coverlet.Core.Tests.Instrumentation
 {
     public class ModuleTrackerTemplateTestsFixture : IDisposable
     {
+        // Prevent parallel execution of tests using the ModuleTrackerTemplate static class
```

Comment:
❔ Why not mark the tests as being in a collection that doesn't allow parallel execution? The semaphore works, but it's not the standard xUnit approach so it suffers from decreased clarity.

Reply:
Brilliant, I didn't know about that feature. I'll change to use that.

Reply:
Fixed